### PR TITLE
Reclassifies Dominia sashes as generic coloured sashes, adds colourable sash variant

### PR DESCRIPTION
**Pull request by Jaraci**
There's a neat bunch of sashes in the loadout section under accessories, but despite being fairly generic sprites they're formally classified as Dominian wear. Dominia already has about a billion different clothing options specific to it strewn across every section of the loadout (to be exact, not counting the sashes there are still _fifteen_ loadout buttons you can press related to Dominia or the Moroz Tribunal) so I reclassified these sashes as more generic items. They can still be used as specifically Dominian sashes through use of the custom item naming/description feature; they just don't default as such anymore.

In addition, I added a colourable variant of the sash sprite in a similar manner to how armbands work. Thanks to @NonQueueingMatt for help with figuring out how to make a recolourable loadout item.

EDIT: This change is, additionally, made with approval from Schwann as the human lore man.

### CODE REVIEW

**Comment by Jaraci** [comment]
!review

**Review by Geevies — APPROVED** [review]
i blame matt if this goes wrong

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -297,21 +297,26 @@
 	gadpathur["medical cadre brassard"] = /obj/item/clothing/accessory/armband/gadpathur/med
 	gear_tweaks += new/datum/gear_tweak/path(gadpathur)
 
-/datum/gear/accessory/dominia/sash
-	display_name = "dominian house sash selection"
-	description = "A selection of sashes representing the Great Houses of Dominia."
-	path = /obj/item/clothing/accessory/dominia/sash
+/datum/gear/accessory/sash_coloured
+	display_name = "sash (colourable)"
+	path = /obj/item/clothing/accessory/sash/colourable
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:302]: bri'ish detected

**Review by JohnWildkins — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (Jaraci) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:302]: this is consistent with the colourable armband :(

**Pull request merged**